plugins/jobs/src/js/JobsOverview.tsx

```diff
@@ -0,0 +1,153 @@
+import * as React from "react";
+import Router, {
+  routerShape,
+  InjectedRouter,
+  LocationDescriptor
+} from "react-router";
+
+// tslint:disable-next-line:no-submodule-imports
+import { BehaviorSubject } from "rxjs/BehaviorSubject";
+// tslint:disable-next-line:no-submodule-imports
+import "rxjs/add/observable/empty";
+
+import { getContext } from "recompose";
+import { graphqlObservable, componentFromStream } from "data-service";
+import gql from "graphql-tag";
+
+// import JobsTabLoading from "./components/JobsTabLoading";
+// import JobsTabError from "./components/JobsTabError";
+// import JobsTabList from "./components/JobsTabList";
+// import JobsTabEmpty from "./components/JobsTabEmpty";
+
+import JobModel from "./data/JobModel";
+import { combineLatest } from "rxjs/observable/combineLatest";
+import { Observable, Subscribable } from "rxjs/Observable";
+import { JobConnection } from "#PLUGINS/jobs/src/js/types/JobConnection";
+import { Job } from "#PLUGINS/jobs/src/js/types/Job";
+
+interface JobsOverviewProps {
+  path: string[];
+  router: InjectedRouter;
+  location: LocationDescriptor;
+  params: Router.Params;
+}
+
+const withRouter = getContext({
+  router: routerShape
+});
+
+function DummyComponent(props: any): JSX.Element {
+  // tslint:disable-next-line:no-console
+  console.log("dummy with props", props);
+
+  function updateFilter(event: React.FormEvent<HTMLInputElement>) {
+    props.handleFilterChange(event.currentTarget.value);
+  }
+
+  return (
+    <div style={{ background: "white" }}>
+      <p>Path</p>
+      {props.data && props.data.path.join("/")}
+      <p>Filter</p>
+      <input value={props.filter} onChange={updateFilter} />
+      <p>Jobs</p>
+      {props.data &&
+        props.data.nodes.map((job: Job) => <p key={job.id}>{job.name}</p>)}
+      <p>Children</p>
+      <div>{props.children}</div>
+    </div>
+  );
+}
+
+const JobsOverview = withRouter(
```

Comment:
I found this which does what our self-written component does, but navigating back in browser history doesnt work currently :(

Reply:
Looks like this is only broken for "index" (`/jobs/overview2`) route 🤔